### DISCUSSION TITLE
Skroutz Analytics for WooCommerce - ydroartas.gr

### DISCUSSION THREAD

**Discussion by entiposis** [community/community#114353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm having an issue with the "Skroutz Analytics for WooCommerce" plugin, Skroutz and our website give different prices.
We asked Skroutz and as I understand it is something a developer needs to see.

The answer from Skroutz is as follows:
"Good evening from Skroutz,

following your message, we would like to inform you that the prices are drawn exclusively from the XML file of your store. The price of 18 euros is currently sent to the file https://prnt.sc/nmho9N9pI7k0 for the product in question.
At the same time, following a check carried out, differences are found between the version of the XML file of your server and the one received by our platform. Specifically, it was found that your XML file has not been modified in the last automatic updates (scans). Please confirm with your developer that the XML file served to the Skroutz bot is updated correctly and that the update of the file is not blocked by other mechanisms (eg caching). https://developer.skroutz.gr/el/feedspec/#skroutzbot"

Any help?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @entiposis ! 

The Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects.

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on XML. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
